addons/website/views/snippets/s_numbers_list.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_numbers_list" name="Numbers list">
+    <section class="s_numbers_list o_cc o_cc1 pt80 pb80">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-5">
+                    <h3>Key Metrics of Company's Achievements</h3>
+                    <p><br/>From revenue growth to customer retention and market expansion, our key metrics of company achievements underscore our strategic prowess and dedication to driving sustainable business success.</p>
+                </div>
+                <div class="col-lg-6 offset-lg-1 s_col_no_bgcolor" data-name="Numbers group">
+                    <div class="row">
+                        <div class="col-lg-6 pt16 pb16">
```

Comment:
```suggestion
                        <div class="col-6 col-lg-6 pt16 pb16">
```
Looks like the perfect occasion to use mobile columns 😉

![image](https://github.com/user-attachments/assets/c4fefae9-ab13-4ad7-91b4-39a8ef9cf716)